### PR TITLE
First version

### PR DESCRIPTION
**Pull request by elisabettai**
First version, checked with TN that this fits his purpose for the moment. We'll get feedback from the testers on the UX.

In the meantime, I have added a couple of TODOs in the code to make it work more nicely,

### CODE REVIEW

**Review by pcrespov — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+		sh -c "cd /sct_label_utils && ooil compose"
+
+.PHONY: build
+build: compose-spec ## build docker images
+	docker-compose build
+
+#.PHONY: run-local
+run-local: ## runs images with local configuration. TODO: make application start with inputs provided. Not it just waits for the input
+	rm -f sct_label_utils.zip
+	smbget -U ${DEVOPS_USER}%${DEVOPS_PASSWORD} smb://biobackup.speag.com/osparc/data/sct_label_utils/sct_label_utils.zip -o sct_label_utils.zip
```
Comment (GitHK) [inline Makefile:41]: This smb get command is a bit strange. If you require data you should put it somewhere publicly available

File: app/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+EXPOSE 12000
+EXPOSE 8080
+
+ENV LC_ALL=C.UTF-8
+ENV LANG=C.UTF-8
+
+FROM base AS production
+
+# TODO: Change according to you needs.
```
Comment (GitHK) [inline app/Dockerfile]: maybe this todo is no longer required?

File: app/config/start.sh [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    do 
+    sleep 1
+    echo "$INFO: Waiting for image file in first input port"
+    done
+
+while [ ! -f ${DY_SIDECAR_PATH_INPUTS}/input_2/labels.txt ]
+    do 
+    sleep 1
+    echo "$INFO: Waiting for input labels txt file in second input port"
+    done
```
Comment (GitHK) [inline app/config/start.sh:20]: In the future you might also want to validate the content of this file. That way you can hint users on what is happening.

File: app/config/supervisord.conf [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+autorestart=true
+stdout_logfile=/dev/fd/1
+stdout_logfile_maxbytes=0
+redirect_stderr=true
+
+[program:app]
+priority=1
+environment=DISPLAY=:0
+directory=/home/scu/work
+# TODO: Change according to you needs
```
Comment (GitHK) [inline app/config/supervisord.conf]: this comment should no longer be required

**Review by GitHK — APPROVED** [review]
some minor stuff below

File: app/config/start.sh (hunk 3 above)
Comment (elisabettai) [reply app/config/start.sh:20]: Indeed. In the future I would like to have this second input provided as a list of integers in the input port (and validated by the content schema, I have a TODO for that in .osparc/web/metadata.yml)

**Pull request merged**